seq_client.go

```
@@ -33,16 +34,16 @@ func (sc *SeqClient) Send(event *SeqLog, apiKey string) bool {
 	if err != nil {
 		log.Fatal(err)
 	}
-
-	client := &http.Client{}
-
 	response, err := client.Do(request)
-
+	defer request.Body.Close()
+	if err != nil {
+		fmt.Println(err)
```

I think it's better the function return an error instead of bool, and return the error here instead of printing it